server/src/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -144,6 +145,8 @@
     @Inject
     SnapshotDataStoreDao _snapshotStoreDao;
     @Inject
+    private DataStoreManager _dataStoreMgr;
```

Comment:
Thanks for finding that, @DaanHoogland! I have corrected it and I will rebase against master now for @swill.